lib/OpenQA/Scheduler.pm

```diff
@@ -56,12 +56,12 @@ our (@ISA, @EXPORT, @EXPORT_OK, %EXPORT_TAGS);
 @ISA = qw(Exporter);
 
 @EXPORT = qw(worker_register worker_get workers_get_dead_worker list_workers job_create
-  job_get job_get_by_workerid jobs_get_dead_worker list_jobs job_grab job_set_done
+  job_get job_get_by_workerid jobs_get_dead_worker list_jobs
```

Comment:
Hmm, I wonder what this @EXPORT is for anyway - we use Scheduler::list_jobs anyway. But list_jobs is still used in api/v1/job - and we better don't touch the json export to avoid API breakage.